dataproc_jupyter_plugin/controllers/editDagController.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
```

Comment:
I think this should be combined with the `executorController.py` file, since it's specifically for executing notebooks (I believe).

If my understanding is right, then move the handlers from this file and the `executorController.py` file into a single file just named `executor.py`.

Then, combine the `dataproc_jupyter_plugin/services/editDagService.py` and `dataproc_jupyter_plugin/services/executorService.py` file into a single file under `dataproc_jupyter_plugin/services` named simply `executor.py`.

That one file will become quite large, but I think the right way to deal with that is not to split it into different files under `services`, but rather to extract out some of the helper methods into a package under `dataproc_jupyter_plugin/executor`. However, that bit is optional and can wait until later.